client/service/src/builder.rs

```diff
@@ -982,6 +983,10 @@ ServiceBuilder<
 				"disk_read_per_sec" => info.usage.as_ref().map(|usage| usage.io.bytes_read).unwrap_or(0),
 				"disk_write_per_sec" => info.usage.as_ref().map(|usage| usage.io.bytes_written).unwrap_or(0),
 			);
+			#[cfg(not(target_os = "unknown"))]
```

Comment:
Why is that now needed?

Reply:
Because if I did that, it would have panicked when you started the light client. We now need to do it this way because you can't call `parity_util_mem::malloc_size` on wasm because `wasm_timer::wasm::Instant` doesn't impl `MallocSizeOf`.